tests/e2e/ligo-plugin-e2e-tests.spec.ts

```diff
@@ -237,4 +237,80 @@ describe('Ligo Plugin E2E Testing for Taqueria CLI', () => {
+		expect(stdout1).toMatchInlineSnapshot(`
+              [
+                "┌───────────────────────────┬───────────────────────────────────────────────┐",
+                "│ Contract                  │ Artifact                                      │",
+                "├───────────────────────────┼───────────────────────────────────────────────┤",
+                "│ counter.mligo             │ artifacts/counter.tz                          │",
+                "├───────────────────────────┼───────────────────────────────────────────────┤",
+                "│ counter.storageList.mligo │ artifacts/counter.default_storage.tz          │",
+                "│                           │ artifacts/counter.storage.another_count.tz    │",
+                "├───────────────────────────┼───────────────────────────────────────────────┤",
+                "│ counter.parameters.mligo  │ artifacts/counter.parameter.increment_by_3.tz │",
+                "└───────────────────────────┴───────────────────────────────────────────────┘",
+              ]
+          `);
+
+		await execute('taq', 'install ../taqueria-plugin-flextesa', './test-project');
+		await exists('./test-project/node_modules/@taqueria/plugin-flextesa/index.js');
+
+		const { stdout: stdout2 } = await execute('taq', 'start sandbox local', './test-project');
+		expect(stdout2).toEqual(expect.arrayContaining(['Starting node...']));
+
+		const { stdout: stdout3 } = await execute('taq', 'list accounts local', './test-project');
+		expect(stdout3).toEqual(expect.arrayContaining(['│ Account │ Balance │ Address                              │']));
+		expect(stdout3).toEqual(expect.arrayContaining([expect.stringContaining('bob')]));
+		await cleanup();
+	});
+
+	test('ligo compile will error with missing storage file', async () => {
+		const { execute, cleanup, writeFile, exists, ls } = await prepareEnvironment();
+		await execute('taq', 'init test-project');
+		await exists('./test-project/.taq/config.json');
+
+		await execute('taq', 'install ../taqueria-plugin-ligo', './test-project');
+		await exists('./test-project/node_modules/@taqueria/plugin-ligo/index.js');
+
+		const mligo_file = await (await exec(`cat e2e/data/ligo-data/hello-tacos.mligo`)).stdout;
+		await writeFile('./test-project/contracts/hello-tacos.mligo', mligo_file);
+		const parameters_file = await (await exec(`cat e2e/data/ligo-data/hello-tacos.parameters.mligo`)).stdout;
+		await writeFile('./test-project/contracts/hello-tacos.parameters.mligo', parameters_file);
+		expect(await ls('./test-project/contracts')).toEqual(
+			expect.arrayContaining(['hello-tacos.mligo', 'hello-tacos.parameters.mligo']),
+		);
+
+		await execute('taq', 'compile hello-tacos.mligo', './test-project');
+		expect.arrayContaining(['│ hello-tacos.parameters.mligo │                          │']),
+			await execute('taq', 'install ../taqueria-plugin-flextesa', './test-project');
```

Comment:
Lines 305-through-313 can be removed, right?

Reply:
removed and correct replacement made

thanks